src/compiler/core.ts

```diff
@@ -1926,9 +2185,10 @@ namespace ts {
             return false;
         }
 
+        // File-system comparisons should use predictable ordering

```

Comment:
This comment isn't going to make sense in this context unless you mention that this is contrasted with using locale-sensitive operations.